osgi-test/build.gradle

```diff
@@ -66,6 +66,8 @@ test {
   systemProperty 'ehcache.osgi.jar', project(':dist').jar.archivePath.getPath()
   systemProperty 'ehcache.osgi.jcache.version', parent.jcacheVersion
   systemProperty 'ehcache.osgi.slf4j.version', parent.slf4jVersion
+}.doFirst {
```

Comment:
I couldn't get this to work... but I didn't dig too hard.  Nothing is missing here except test coverage though.  I think the best thing to do would be to file an issue to try and fix the test harness (if that is possible).